testsuite/pom.xml

```diff
@@ -142,6 +142,21 @@
                 <version.resteasy.testsuite>${version.resteasy.testsuite}</version.resteasy.testsuite>
             </properties>
         </profile>
+
+        <!-- https://issues.jboss.org/browse/MODULES-372 - JDK 11 - module not found java.se -->
+        <profile>
+            <id>modular-jdk</id>
+            <activation>
+                <jdk>[9,)</jdk>
```

Comment:
Is it really Java 9+ or 11+ ?

Reply:
We in sync with WF, https://github.com/wildfly/wildfly/blob/master/pom.xml#L6985-L6997